src/main/java/technology/semi/weaviate/client/v1/graphql/GraphQL.java

```diff
@@ -47,6 +48,10 @@ public Get get() {
     return new Get(config);
   }
 
+  public RawGQL rawGQL() {
```

Comment:
I would rename this to `raw` just to be inline with `python client`

Reply:
I would here make this method `raw(String query)` and there just return the interface that already runs the query. I think there's no need for yet another layer of execution, wdyt @aliszka ?

Reply:
example:
```
Result<GraphQLResponse> result = client.graphQL().raw("{Get{Pizza{_additional{id}}}}").run();
```